packages/edit-site/src/components/page-patterns/utils.js

```diff
@@ -1,21 +1,2 @@
-export const DEFAULT_CATEGORY = 'my-patterns';
-export const DEFAULT_TYPE = 'wp_block';
-export const PATTERNS = 'pattern';
-export const TEMPLATE_PARTS = 'wp_template_part';
-export const USER_PATTERNS = 'wp_block';
-export const USER_PATTERN_CATEGORY = 'my-patterns';
```

Comment:
Was there a reason to split USER and DEFAULT even though they have the same value? If so I can re-split.